dallinger/experiment.py

```diff
@@ -187,6 +187,67 @@ def recruiter(self):
         """
         return recruiters.from_config(get_config())
 
+    @property
+    def qualifications(self):
+        """All the qualifications we want to assign to a worker.
+
+        By default, workers will always be assigned one qualification specific
+        to the experiment run. The property `group_qualifications` defines
+        additional qualifications to assign.
+
+        Type should be a list of dictionaries with "name", "description", and
+        optionally "score" (an integer), or an empty list.
+
+        If no "score" is provided, the recruiter will increment the worker's
+        current score for the qualification, setting it to 1 if it doesn't
+        previously exist.
+        """
+        experiment_qualification_desc = "Experiment-specific qualification"
+        config = get_config()
+        quals = [
+            {
+                "name": config.get("id"),
+                "description": experiment_qualification_desc,
+            }
+        ]
+
+        return quals + self.group_qualifications
```

Comment:
My rationale was that you're unlikely to want to override the experiment ID-based qualification, and are more likely to override the group qualifications (mostly obviously, not using the `group_name` config variable, but just defining the qualifications in the property instead), but I agree it's gotten complicated and may not be worth it.

Reply:
Right that makes sense, but if both methods have default implementations that would need to be replicated in a sub-class it seems like we're making things more complicated than they need to be. It seems like a `qualifications()` method would suffice, especially since most use cases would just add some additional qualifications to the list pulled in with a `super()` call. A property seems like an unnecessary complication that makes documenting it more difficult. Is there a use case where this wouldn't be dynamic and someone could just set the attribute on the class?

Reply:
> Is there a use case where this wouldn't be dynamic and someone could just set the attribute on the class?
That's what I had in mind, based on a vague recollection of someone wanting to do that (set class attributes for this kind of thing).

Still, I think we need to fight back complication where possible, so I'm inclined to reduce this all to a single method as you suggest. I'll ask the group tomorrow. <reminder: @pmcharrison @jacobyn @skleinfeldt >